pkgs/development/python-modules/brevo-python/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  certifi,
+  python-dateutil,
+  six,
+  urllib3,
+}:
+
+buildPythonPackage rec {
+  pname = "brevo-python";
+  version = "1.1.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "getbrevo";
+    repo = "brevo-python";
+    rev = "v${version}";
+    hash = "sha256-XOUFyUrqVlI7Qr4uzeXr6GJuQ+QTVhsueT1xxVQMm14=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    certifi
+    python-dateutil
+    setuptools
+    six
+    urllib3
+  ];
+
+  pythonImportsCheck = [ "brevo_python" ];
+
```

Comment:
Can you make the `doCheck = false` explicit?

Reply:
```suggestion
  nativeCheckInputs = [
    pytestCheckHook
  ];

  pythonImportsCheck = [
    "brevo_python"
  ];
```